src/renderers/canvas/sigma.canvas.labels.def.js

```diff
@@ -24,7 +24,8 @@
         labelWidth,
         labelOffsetX,
         labelOffsetY,
-        alignment = settings('labelAlignment');
+        alignment = settings('labelAlignment'),
+        maxLineLength = settings('maxNodeLabelLineLength') || 0;
```

Comment:
`maxNodeLabelLineLength` should be added to the sigma settings with default value to `0`. Currently it is  not possible to discover this setting but reading at the source code.